tritonsrc/attn_torch_function.py

```diff
@@ -608,6 +626,10 @@ def backward(ctx, do, _, fwd_tuning_result):
                     PADDED_HEAD=padded_head,
                     BIAS_TYPE=ctx.bias_type,
                 )
+        print(f"{dq.stride()=}", flush=True)
```

Comment:
Nit, maybe conditionalize prints in general or use one of the built in loggers.  If done, work for another PR.

Reply:
Will remove this in the bwd kernel PR.